topo/node/ceos/ceos.go

```diff
@@ -248,10 +302,10 @@ func defaults(pb *tpb.Node) *tpb.Node {
 	}
 	if pb.Labels == nil {
```

Comment:
this can be simplified:

if pb.Labels == nil {
  pb.Labels = map[string]string{}
}
if pb.Labels["type"] == "" {
  pb.Labels["type"] = tpb.Node_ARISTA_CEOS.String()
}
if pb.Labels["vendor"] == "" {
  pb.Labels["vendor"] = tpb.Vendor_ARISTA.String()
}
if pb.Labels["model"] == "" {
  pb.Labels["model"] = pb.Model
}
if pb.Labels["os"] == "" {
  pb.Labels["os"] = pb.Os
}
if pb.Labels["version"] == "" {
  pb.Labels["version"] = pb.Version
}

Reply:
+1